src/main/kotlin/no/nav/syfo/persistering/db/PersisterDialogmeldingQueries.kt

```diff
@@ -372,6 +372,91 @@ fun DatabaseInterface.getUnpublishedArenaMeldinger(): List<Triple<String, String
         }
     }
 
+fun DatabaseInterface.getAdhocArenaMeldinger(): List<Triple<String, String, String>> =
+    connection.use { connection ->
+        connection.prepareStatement(
+            """
+            SELECT id, fellesformat, msg_id
+            FROM dialogmeldingopplysninger d
+            WHERE !sent_to_arena
```

Comment:
Funker det med `!` eller må man bruke `not` ?

Reply:
Fikk akkurat feil på det i dev - har fikset nå.